Competition2018/src/main/java/competition/subsystems/elevator/commands/CalibrateElevatorViaStallCommand.java

```diff
@@ -15,15 +16,21 @@
 
     DoubleProperty power;
     DoubleProperty calibrationTime; // In milliseconds
+    DoubleProperty calibrationCurrentThreshold;
+    DoubleProperty goodCurrentRange;
 
     double targetTime;
     boolean atTarget;
 
+
     @Inject
     public CalibrateElevatorViaStallCommand(XPropertyManager propMan, CommonLibFactory clf) {
```

Comment:
For this command, you only need to measure the peak current and see if it is above a key threshold.

Reply:
"feed" means I need to change the power by tracking the current?

Reply:
delete it!